splice_machine/src/main/java/com/splicemachine/derby/stream/function/NLJoinFunction.java

```diff
@@ -95,6 +112,24 @@ private void initOperationContexts() throws StandardException {
         }
     }
 
+    @Override
+    public void close() {
+        StandardException se = null;
+        for (OperationContext ctx : activeContexts) {
+            try {
+                ctx.getOperation().close();
+            } catch (StandardException e) {
+                se = e;
+                LOG.error("Exception while closing operation", e);
+            }
+        }
```

Comment:
should we close currentOperationContext here too? From the code in loadBatch(), it could be removed from the activeContexts, so it may not be in the activeContexts when close() is called.

Reply:
With the new change, closing all contexts should take care of that.